platform/saivpp/vpplib/SwitchStateBaseRif.cpp

```diff
@@ -855,12 +861,57 @@ sai_status_t SwitchStateBase::process_interface_loopback (
     SWSS_LOG_NOTICE("interfaceName:%s isLoopback:%u", interfaceName.c_str(), isLoopback);
 
     if (isLoopback) {
-        vpp_add_del_lpb_intf_ip_addr(serializedObjectId, is_add);
+        std::string vppIfName = "loop" + std::to_string(getInstanceFromHostIfname(interfaceName));
+
+        if (is_add && (lpbInstMap.find(vppIfName) != lpbInstMap.end()))
+        {
+            // interface already exists - store dual stack IP
+            lpbIpToIfMap[destinationIP] = vppIfName;
+            lpbIpToHostIfMap[destinationIP] = interfaceName;
+            SWSS_LOG_DEBUG("interfaceName:%s exists new-ip:%s",
+                interfaceName.c_str(), destinationIP.c_str());
```

Comment:
Aren't you supposed to configure the IPv6 address in the VPP ? You are just returning here without adding v6 address. This fix will not work.

Reply:
Looks like the address update happens as part of the existing code itself. Anyhow, I added a function to update the address as you suggested.